### PR TITLE
feat(learn): Stock Price Checker user stories

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related #39721 

* Modifies verbiage to use second person
* Adds standard description
* Brings instructions over from boilerplate readme

### CODE REVIEW

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@nhcarrigan When I look at the boilerplate, I can not tell how many tests there need to be.  There are not `test('test text', function(done) { })` as there are in other boilerplates.  Can the boilerplates be changes to explain what test to write like is done in PR https://github.com/freeCodeCamp/freeCodeCamp/pull/39976 ?

**Comment by nhcarrigan** [comment]
Should be fixed 🙂

File: curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.md [hunk 1]
```diff
@@ -9,40 +9,50 @@ forumTopicId: 301572
...
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but must be publicly visible for our testing.
-Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-stockchecker">this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-stockchecker/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
+
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-stockchecker/'>this GitHub repo</a> and complete your project locally.
```
Comment (RandellDawson) [inline curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.md:14]:
```suggestion
- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-stockchecker/' target='_blank'>this GitHub repo</a> and complete your project locally.
```

**Review by moT01 — DISMISSED** [review]
I think I'm fine with this - LGTM 🎉

File: curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.md [hunk 2]
```diff
@@ -9,40 +9,58 @@ forumTopicId: 301572
...
       "
-  - text: I can GET /api/stock-prices with form data containing a Nasdaq stock ticker and receive back an object stockData.
+  - text: You can send a `GET` request to `/api/stock-prices` with form data containing a NASDAQ stock ticker. Returned will be an object with the property `stockData`.
     testString: "async getUserInput => {
       const data = await fetch(getUserInput('url') + '/api/stock-prices?stock=GOOG');
       const parsed = await data.json();
       assert.property(parsed, 'stockData');
     }"
-  - text: In stockData, I can see the stock (the ticker as a string), price (decimal in string format), and likes (int).
+  - text: The `stockData` property includes the `stock` ticker as a string, the `price` (decimal in string format), and `likes` as a number.
```
Comment (Sky020) [inline curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.md:63]:
```suggestion
  - text: The `stockData` property includes the `stock` ticker as a string, the `price` as a number, and `likes` as a number.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This does not have to do with your PR, but is an issue with the tests/user story not matching.

**Review by ShaunSHamilton — APPROVED** [review]
Thanks, Nicholas. LGTM 👍 

@moT01 Tom, if you are still happy with this, I am.

**Review by RandellDawson — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Side note:
Seems like we could use a heading on most of these projects here...
Screen Shot 2020-11-25 at 11 26 59 AM

It kind of abruptly gets into the instructions - maybe simply `Instructions` or `Complete the following in the boilerplate:`

Certainly something for another issue though - maybe I'll make one. Thanks for the work and reviews on this everyone 🎉

**Pull request merged**